lib/manageiq/postgres_ha_admin/config_handler/logical_replication_config_handler.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+require 'pg/dsn_parser'
+
+module ManageIQ
+module PostgresHaAdmin
```

Comment:
why the weird module indents?

Reply:
I think that's been there since we extracted the gem. It's that way in places in the [console as well](https://github.com/ManageIQ/manageiq-appliance_console/blob/a31bba6b6c6e87cf71a9be6fc1cc39841e6052f4/lib/manageiq/appliance_console/database_replication.rb#L5-L6)